### PR TITLE
Ltscloud 1094

### PR DESCRIPTION
**Pull request by cgoines**
this is to fix libcloud-1094

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,8 +57,8 @@
   <!--  aws jars -->
    <dependency>
      <groupId>com.amazonaws</groupId>
-     <artifactId>aws-java-sdk</artifactId>
-     <version>${aws-java-sdk.version}</version>
+     <artifactId>aws-java-sdk-bundle</artifactId>
+     <version>1.12.1</version>
```
Comment (michael-lts) [inline pom.xml:61]: Change this back to ${aws-java-sdk.version}

**Review by michael-lts — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -88,20 +89,27 @@
...
+      <version>1.2.21</version>
+    </dependency>
+
+<!--
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
+    </dependency>
+-->
```
[the remark this answers is not in the document]
Comment (cgoines) [reply pom.xml:105]: theres another dependency right after that. do you want to keep that?

File: pom.xml [hunk 3]
```diff
@@ -16,12 +16,13 @@
...
     <camel.version>2.17.7</camel.version><!-- last release for Java 1.7-->
-    <!-- <aws-java-sdk.version>1.11.301</aws-java-sdk.version> -->
-    <aws-java-sdk.version>1.10.8</aws-java-sdk.version>
+    <aws-java-sdk.version>1.12.1</aws-java-sdk.version>
+    <!-- <aws-java-sdk.version>1.10.8</aws-java-sdk.version> -->
     <io.jsonwebtoken.version>0.10.7</io.jsonwebtoken.version>
     <resteasy.version>3.5.0.Final</resteasy.version>
     <junit.jupiter.version>5.2.0</junit.jupiter.version>
     <gson.version>2.8.2</gson.version>
+    <aws-java-sdk.version>1.10.8</aws-java-sdk.version>
```
[the remark this answers is not in the document]
Comment (cgoines) [reply pom.xml:25]: done

File: pom.xml [hunk 4]
```diff
@@ -16,12 +16,13 @@
...
     <camel.version>2.17.7</camel.version><!-- last release for Java 1.7-->
-    <!-- <aws-java-sdk.version>1.11.301</aws-java-sdk.version> -->
-    <aws-java-sdk.version>1.10.8</aws-java-sdk.version>
+    <aws-java-sdk.version>1.12.1</aws-java-sdk.version>
+    <!-- <aws-java-sdk.version>1.10.8</aws-java-sdk.version> -->
     <io.jsonwebtoken.version>0.10.7</io.jsonwebtoken.version>
     <resteasy.version>3.5.0.Final</resteasy.version>
     <junit.jupiter.version>5.2.0</junit.jupiter.version>
     <gson.version>2.8.2</gson.version>
+    <aws-java-sdk.version>1.12.1</aws-java-sdk.version>
```
Comment (michael-lts) [inline pom.xml:25]: Looks like this is a repeat of line 19

**Review by michael-lts — CHANGES_REQUESTED** [review]

**Review by michael-lts — APPROVED** [review]
All requests for cosmetic changes done, and this has been tested in qa

**Pull request merged**